quesma/model/expr.go

```diff
@@ -13,7 +13,8 @@ var InvalidExpr = Expr(nil)
 
 // ColumnRef is a reference to a column in a table, we can enrich it with more information (e.g. type used) as we go
 type ColumnRef struct {
-	ColumnName string
+	OptPrefixTable string // used for alias in joins, mostly empty
```

Comment:
This can be named "TableAlias". The question is how we should mark optional elements. The pointer can cause nil panics.

Reply:
Renamed, not sure if this is best, but solves my issue. Optional elements are usually pointers, but here empty string seems nicer.